### PR TITLE
Implement Apache Solr as optional feature

### PR DESCRIPTION
**Pull request by stefnats**
As discussed yesterday in PR #1163 this is now for Homestead 9.

To activate Solr just use in Homestead.yaml:

    features:
        - solr: true

And open the port on:

    ports:
        - send: 18983
          to: 8983

This installs Solr 7.7.1 per default, current version is 8.1.0 (as of today). I have not much experience with the 8.* branch of Solr, so i implemented the latest 7.* version. It also installs a default "homestead" search core.

-Stefan

### CODE REVIEW

File: scripts/features/solr.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Install Java Runtime Enviroment
+sudo apt-get update
```
Comment (browner12) [inline scripts/features/solr.sh]: @svpernova09 can correct me if I'm wrong, but I believe we decided on switching to `apt` instead of `apt-get` for v9.

**Comment by stefnats** [comment]
Didn't know that! Corrected.

File: scripts/features/solr.sh (hunk 1 above)
Comment (svpernova09) [reply scripts/features/solr.sh]: I'm somewhat torn, while I would like to switch to apt, and we currently have in V9, even apt gives CLI warnings that the CLI interface isn't stable, which has me rethinking it.

**Comment by svpernova09** [comment]
Thanks!

File: scripts/features/solr.sh (hunk 1 above)
Comment (browner12) [reply scripts/features/solr.sh]: interesting answer here. https://askubuntu.com/questions/990823/apt-gives-unstable-cli-interface-warning

maybe we were wrong on this.  sounds like we should use things like `apt-get` and `apt-cache` for scripts, but users running straight from the command line should use `apt`.

**Pull request merged**